release-tools/release-aux/release-data-fn.sh

```diff
@@ -8,8 +8,8 @@
 
 # Public or premium release?  Let the version numbers determine it!
 declare -A _ossl_release_types=(
-    [premium]='^1\.0\.2'
-    [public]='^(1\.1\.1|[3-9]\.)'
+    [premium]='^1\.[01]\.[12]'
```

Comment:
This to me (I think) says premium releases are identified as 1.0.1, 1.0.2, 1.1.1 and 1.1.2

Should that be future proofed for a bit by extending [12] to [123456789] or some such?  Presumably, we're going to bump these patch levels, or are we using alphabetic designation in perpetuity here?

Reply:
We IMO should not bump the number as that could possibly break things for our customers depending on the fix being marked by the alphabetic suffix. There will be a problem when we reach 1.0.2zzzzzzzzzze release :grin: as that will be the last one that can be represented by the OPENSSL_VERSION_NUMBER. However that will be reached some time near the end of this century if we keep the current rate of patch releases.